Formula/bazel.rb

```diff
@@ -16,11 +16,14 @@ class Bazel < Formula
 
   def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
+    # Force Bazel to put all temporary file in the homebrew cache
+    ENV["BAZEL_WRKDIR"] = "#{HOMEBREW_CACHE}/bazel-workdir"
```

Comment:
I'm also not sure whether this actually works. I just checked our El Capitan build bot and there are still 1.2 GB that have been dumped to `/var/tmp/_bazel_brew` (I don't think all of it can be attributed to #3426). `~/Library/Caches/Homebrew/bazel-workdir` has been created, but it only contains two completely empty subdirectories …